HdmiInput/HdmiInput.json

```diff
@@ -56,6 +56,8 @@
     },
     "methods": {
         "getHDMIInputDevices": {
+            "deprecated" : true,
```

Comment:
Apart from adding it for individual APIs, can we also add it at the top under Description section. We should add a link to the new plugin (AVInput).

Reply:
Also add deprecated label for all the events.

Reply:
Added deprecation part in Description section. Added deprecated labels for all events